dist/ami/build_ami.sh

```diff
@@ -73,14 +73,25 @@ if [ $LOCALRPM -eq 1 ]; then
         pkg_install git
     fi
 
```

Comment:
I don't understand the goal. What is build_rpms?

Reply:
build_rpm is the script that generates the rpms build_ami uses for the ami when given the `--localrpm` option

Reply:
When `--localrpm` option is assigned for build_ami.sh, it will use build_rpms.sh to build scylla rpms if some scylla packages is missing in ami files directory (dist/ami/files/).